photosynthesis_metrics/perceptual.py

```diff
@@ -0,0 +1,280 @@
+"""
+Implementation of VGG16 loss, originaly used for style transfer and usefull in many other task (including GAN training)
+"""
```

Comment:
Since we have references to initial papers implementations in other metrics, it would be great to also add it here